core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/PoolOrderCompletion.java

```diff
@@ -129,6 +131,10 @@ public Date getLastUpdated() {
     return lastUpdated;
   }
 
+  public int getLoaded() {
```

Comment:
GitHub is missing party parrot emoji. 🙁 Can you add javadoc to clarify what this means?